apps/web/public/static/locales/en/common.json

```diff
@@ -1481,6 +1481,8 @@
   "report_app": "Report app",
   "limit_booking_frequency": "Limit booking frequency",
   "limit_booking_frequency_description": "Limit how many times this event can be booked",
+  "limit_booking_only_first_slot": "Limit booking only first slot",
```

Comment:
note for later: improve wording, maybe something like

"Limit booking to the next available slot of a day."